add-domain

```diff
@@ -768,3 +768,5 @@ soclaiemx.xyz
 wiflix.xyz
 wiflix.yachts
 coronavirus.zone
+gіthսᖯ.com
```

Comment:
is == `xn--gth-jhd19l0y3c.com`

Reply:
Update.

This domain is no longer active

```
xn--gth-jhd19l0y3c.com                                                                               INACTIVE    STDLOOKUP  Unknown           Unknown    AVAILABILITY
```

Reply:
yeah, it was not active before. I think I should just remove it from the PR. I believe I should only add registered **homograph domains** in the Pull Requests because if I add all possible combinations, I will flood this repository with several thousands of domains :)

Reply:
I just pushed another change to this PR, and added the two registered homograph domains that are similar to google.com

Reply:
I working on your **matrix** repo right now